kallisto/urls.py

```diff
@@ -7,6 +7,7 @@
 urlpatterns = patterns(
     '',
     url(r'^$', 'apps.homepage.views.homepage', name='homepage'),
+    url(r'^help$', 'apps.homepage.views.help', name='help'),
```

Comment:
It's very handy; because `.as_view()` makes a new object (which is a callable, and hence can act as a view), you get the class-set defaults plus anything in the `.as_view()` call. So for instance you can have different URLs with the same underlying code but different details. You can also aggregate them so the variance happens based on something else you can detect early on, like HTTP headers.

This is supposed to be have a [helpful example](https://docs.djangoproject.com/en/1.8/topics/class-based-views/mixins/#an-alternative-better-solution) in the documentation, but that's unclear and confusing :-(

(The [ref docs](https://docs.djangoproject.com/en/1.8/ref/class-based-views/base/#django.views.generic.base.View.as_view) for `.as_view()` don't help either.)